commit 2b964c84dba0edc15fa81302aae366a2b8e3b943
Author: Joel E. Denny <jdenny.ornl@gmail.com>
Date:   Thu Sep 7 12:34:49 2023 -0400

    Revert "[lit] Fix f254bbf23374 FileCheck patterns"
    
    This reverts commit 3db5db92d746bad8ba1762ca290a176e25d48565.
    
    The reason for the revert is discussed at:
    https://discourse.llvm.org/t/rfc-improving-lits-debug-output/72839/52

diff --git a/llvm/utils/lit/tests/shtest-env.py b/llvm/utils/lit/tests/shtest-env.py
index c093b62e5420..e7c64e2ae8af 100644
--- a/llvm/utils/lit/tests/shtest-env.py
+++ b/llvm/utils/lit/tests/shtest-env.py
@@ -85,7 +85,7 @@
 
 # CHECK: FAIL: shtest-env :: env-calls-not-builtin.txt ({{[^)]*}})
 # CHECK: env -u FOO BAR=3 not rm {{.+}}.no-such-file
-# CHECK: # executed command: env -u FOO BAR=3 not rm {{.+}}.no-such-file{{.*}}
+# CHECK: # executed command: env -u FOO BAR=3 not rm {{.+}}.no-such-file{{.+}}
 # CHECK: # | Error: 'env' cannot call 'rm'
 # CHECK: # error: command failed with exit status: {{.*}}
 
diff --git a/llvm/utils/lit/tests/shtest-not.py b/llvm/utils/lit/tests/shtest-not.py
index f514cddf1def..36cc0e60d3f3 100644
--- a/llvm/utils/lit/tests/shtest-not.py
+++ b/llvm/utils/lit/tests/shtest-not.py
@@ -106,7 +106,7 @@
 
 # CHECK: FAIL: shtest-not :: not-calls-env-builtin.txt {{.*}}
 # CHECK: not --crash env -u FOO BAR=3 rm {{.*}}.no-such-file
-# CHECK: # executed command: not --crash env -u FOO BAR=3 rm {{.+}}.no-such-file{{.*}}
+# CHECK: # executed command: not --crash env -u FOO BAR=3 rm {{.+}}.no-such-file{{.+}}
 # CHECK: # | Error: 'env' cannot call 'rm'
 # CHECK: # error: command failed with exit status: {{.*}}
 
diff --git a/llvm/utils/lit/tests/shtest-shell.py b/llvm/utils/lit/tests/shtest-shell.py
index 97d946643b39..52a47f03e53e 100644
--- a/llvm/utils/lit/tests/shtest-shell.py
+++ b/llvm/utils/lit/tests/shtest-shell.py
@@ -238,7 +238,7 @@
 # CHECK-NOT: error
 
 #      CHECK: diff -u {{.*}}.foo {{.*}}.bar | FileCheck {{.*}} && false || true
-# CHECK-NEXT: # executed command: diff -u {{.+}}.foo{{.*}} {{.+}}.bar{{.*}}
+# CHECK-NEXT: # executed command: diff -u {{.+}}.foo{{.+}} {{.+}}.bar{{.+}}
 # CHECK-NEXT: # note: command had no output on stdout or stderr
 # CHECK-NEXT: # error: command failed with exit status: 1
 # CHECK-NEXT: # executed command: FileCheck
@@ -253,8 +253,8 @@
 # CHECK-NOT: error
 
 #      CHECK: cat {{.*}}.bar | diff -u {{.*}}.foo - && false || true
-# CHECK-NEXT: # executed command: cat {{.+}}.bar{{.*}}
-# CHECK-NEXT: # executed command: diff -u {{.+}}.foo{{.*}} -
+# CHECK-NEXT: # executed command: cat {{.+}}.bar{{.+}}
+# CHECK-NEXT: # executed command: diff -u {{.+}}.foo{{.+}} -
 # CHECK-NEXT: # .---command stdout{{-*}}
 #      CHECK: # | @@
 # CHECK-NEXT: # | -foo
@@ -264,8 +264,8 @@
 # CHECK-NEXT: # executed command: true
 
 #      CHECK: cat {{.*}}.bar | diff -u - {{.*}}.foo && false || true
-# CHECK-NEXT: # executed command: cat {{.+}}.bar{{.*}}
-# CHECK-NEXT: # executed command: diff -u - {{.+}}.foo{{.*}}
+# CHECK-NEXT: # executed command: cat {{.+}}.bar{{.+}}
+# CHECK-NEXT: # executed command: diff -u - {{.+}}.foo{{.+}}
 # CHECK-NEXT: # .---command stdout{{-*}}
 #      CHECK: # | @@
 # CHECK-NEXT: # | -bar
@@ -279,8 +279,8 @@
 # CHECK-NOT: error
 
 #      CHECK: cat {{.*}}.bar | diff -u {{.*}}.foo - | FileCheck {{.*}}
-# CHECK-NEXT: # executed command: cat {{.+}}.bar{{.*}}
-# CHECK-NEXT: # executed command: diff -u {{.+}}.foo{{.*}} -
+# CHECK-NEXT: # executed command: cat {{.+}}.bar{{.+}}
+# CHECK-NEXT: # executed command: diff -u {{.+}}.foo{{.+}} -
 # CHECK-NEXT: note: command had no output on stdout or stderr
 # CHECK-NEXT: error: command failed with exit status: 1
 # CHECK-NEXT: # executed command: FileCheck
@@ -427,7 +427,7 @@
 # CHECK: *** TEST 'shtest-shell :: diff-unified.txt' FAILED ***
 
 #      CHECK: diff -u {{.*}}.foo {{.*}}.bar && false || true
-# CHECK-NEXT: # executed command: diff -u {{.+}}.foo{{.*}} {{.+}}.bar{{.*}}
+# CHECK-NEXT: # executed command: diff -u {{.+}}.foo{{.+}} {{.+}}.bar{{.+}}
 # CHECK-NEXT: # .---command stdout{{-*}}
 #      CHECK: # | @@ {{.*}} @@
 # CHECK-NEXT: # | 3
@@ -443,7 +443,7 @@
 # CHECK-NEXT: # executed command: true
 
 #      CHECK: diff -U 2 {{.*}}.foo {{.*}}.bar && false || true
-# CHECK-NEXT: # executed command: diff -U 2 {{.+}}.foo{{.*}} {{.+}}.bar{{.*}}
+# CHECK-NEXT: # executed command: diff -U 2 {{.+}}.foo{{.+}} {{.+}}.bar{{.+}}
 # CHECK-NEXT: # .---command stdout{{-*}}
 #      CHECK: # | @@ {{.*}} @@
 # CHECK-NEXT: # | 4
@@ -457,7 +457,7 @@
 # CHECK-NEXT: # executed command: true
 
 #      CHECK: diff -U4 {{.*}}.foo {{.*}}.bar && false || true
-# CHECK-NEXT: # executed command: diff -U4 {{.+}}.foo{{.*}} {{.+}}.bar{{.*}}
+# CHECK-NEXT: # executed command: diff -U4 {{.+}}.foo{{.+}} {{.+}}.bar{{.+}}
 # CHECK-NEXT: # .---command stdout{{-*}}
 #      CHECK: # | @@ {{.*}} @@
 # CHECK-NEXT: # | 2
